src/polymer-cli.ts

```diff
@@ -122,7 +122,7 @@ export class PolymerCli {
     return Array.from(mergedArgs.values());
   }
 
-  run() {
+  run(): Promise<any> {
```

Comment:
Yeah, and it's an easy thing to add onto a function that already returns a Promise. The only differences in that case are that with async:

  - a default return becomes `Promise.resolve();`
  - a throw in the stack frame of the function become a rejected promise